doc/examples/returning_interaction.rb

```diff
@@ -0,0 +1,30 @@
+# Example: ReturningInteraction
+# As an example we use a ReturningInteraction to update User records in the admin workspace
+
+class UsersController < BaseController
```

Comment:
Let's add the controller and interaction file locations as comments like you do in the next example.

Reply:
I have no idea what would be an appropriate file path for a controller

Reply:
It could be as simple as `app/controllers/users_controller.rb`

and below 

`app/interactions/users/update.rb`